nessie/lakectl_util_test.go

```diff
@@ -0,0 +1,233 @@
+package nessie
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"regexp"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+)
+
+//
+// lakectl tests utility functions
+//
+var update = flag.Bool("update", false, "update golden files with results")
+
+func lakectlLocation() string {
+	return viper.GetString("lakectl_dir") + "/lakectl"
+}
+
+func lakectl() string {
+	os.Setenv("LAKECTL_DIR", "~/Code/lakeFS")
```

Comment:
This may make it hard for me (and others) to run these tests from the commandline.  Since it doesn't really belong here, maybe just take it from the environment, or set it from a test flag?

Reply:
Oops. Bad leftovers (could not find an embarrassed icon...)
To be removed!

Reply:
Done